portal-impl/src/com/liferay/portlet/documentlibrary/messaging/BaseProcessorMessageListener.java

```diff
@@ -38,6 +38,11 @@ protected void doReceive(Message message) throws Exception {
 		catch (Exception exception) {
 			if (_log.isWarnEnabled()) {
 				_log.warn(
+					"Unable to process file version " +
+						destinationFileVersion.getFileVersionId());
+			}
+			else if (_log.isDebugEnabled()) {
```

Comment:
Isn't this redundant? If debugEnabled is true, then warnEnabled will be always true. Unless I'm missing something, can you remove the debug logs when there's also an identical warn one?